src/User32.Desktop/User32+SpecialWindowHandles.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+    using System.Runtime.InteropServices;
+
+    /// <content>
+    /// Contains the <see cref="SpecialWindowHandles"/> nested type.
+    /// </content>
+    public partial class User32
+    {
+        /// <summary>
+        /// Special window handles
+        /// </summary>
+        public enum SpecialWindowHandles
```

Comment:
I agree. Perhaps if enum's can't derive from `IntPtr`, you can change this enum to be a `static class` and make each enum value a `public static IntPtr` field.